packages/e2e-tests/plugins/cpt-locking.php

```diff
@@ -52,6 +52,82 @@ function gutenberg_test_cpt_locking() {
 			'template_lock' => false,
 		)
 	);
+	register_post_type(
+		'ul-post-ul-group',
```

Comment:
If I understood the nomenclature, this should be:

```suggestion
		'l-post-ul-group',
```

Reply:
Correct the string was updated.

Reply:
We are using this nomenclature because of the char limits post type id has.